ux/bolt-CLI-spec.md

```diff
@@ -86,49 +86,96 @@ PARAMETERS:
 
 
 
-**View output per node while task/command/script is running on single or multiple nodes.**
+**View output per node while task/command/script/file is running on single or multiple nodes.**
 
-**--human format (default)**
+**--human format (default) for commands and scripts**
 
 ~~~
-$ bolt [task/command/script] run [options...]
-Starting [task/command/script]...
+$ bolt [command/script] run [options...]
+Starting [command/script]...
 Nodes: 3
 
 Started on node-1...
 Started on node-2...
 Started on node-3...
-Finished on node-1
-  status: [succeeded/failed/<task-specific staus>]
-    STDOUT:
-      [stdout output...]
-    STDERR:
-      [stderr output...]
-Finished on node-2
-  status: [succeeded/failed/<task-specific staus>]
-   STDOUT:
-      [stdout output...]
-    STDERR:
-      [stderr output...] 
-Finished on node-3
-  status: [succeeded/failed/<task-specific staus>]
-    STDOUT:
-      [stdout output...]
-    STDERR:
-      [stderr output...] 
+[Succeeded/Failed] on node-1
+  STDOUT:
+    [stdout output...]
+  STDERR:
+    [stderr output...]
```

Comment:
Should we print
```
STDERR:

```
when there's no stderr output?

Reply:
No. There is a note in the spec: "STDOUT or STDERR labels will be printed only if the output from that stream is non-empty."

Reply:
oops I didn't implement that.
Should any message be printed if there is no output?

Reply:
Let's go with no message for now, unless we learn of a use case where one is required. The status will be printed regardless.